### ISSUE TITLE
Feature Request: Option to not scan subdirectories

### ISSUE THREAD

**Issue by SFrost007** [magneticchen/Daijishou#108]
Daijishou has become my favourite launcher for my Odin, but one thing I can't see is possible right now is to skip subdirectories when scanning roms.

I have a top-level folder containing my "main" roms for each platform, and then subfolders for games I don't want to import into my frontend but would like to have available on my card to load into emulators manually. e.g:
* /roms/gba/Mario.zip
* /roms/gba/Japan/Something.zip

I'd like Daijishou to scan and import all the games in `/roms/gba/*` but **not** `/roms/gba/anything/*`. I tried excluding subdirectory patterns with the regex filter but this didn't work - I guess the regex is only passed the filename, not the path? So either the option to pass the whole file path to the regex filter, or simply a checkbox to enable/disable scanning subdirectories would be great!

**Comment by magneticchen** [comment]
Why not put the root files in to another subfolders 🤔. BTW Dai also allows you to select multiple sync paths.

**Comment by SFrost007** [comment]
> Why not put the root files into another subfolders 🤔

Yeah you're right, I'd thought about this - it would do the trick. But I also have scraped media for all the games as a subfolder of the platform, and worried this might not work if you implement ["Pre-define media folders" from issue 86](https://github.com/magneticchen/Daijishou/issues/86). If it wouldn't cause problems with however that feature is planned (**if** you do work on it) then another subfolder would be fine.

> BTW Dai also allows you to select multiple sync paths.

Yeah this is great! It's how I'd planned to add romhacks along with the "main" roms I had 🙂

**Comment by magneticchen** [comment]
I guess I will maintain the current status that don't add another layer of complexity. And just let the file structure do the works. 👍
